site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -30,6 +30,29 @@
     </div>
 <br><br>
 
+    <div class="option-title">
+        Should this gradeable be locked until a score is reached?
+    </div>
+    <div>
+        <fieldset>
+            <legend>Previous Gradeable:</legend>
+            <div class="option-input col-md-6" style="padding: 0;">
+                <select name="depends_on" id="gradeable-lock">
+                    <option value="">Select an option (Off)</option>
+                    {% for template in template_list %}
+                        <option value="{{ template.g_id }}" {% if template.g_id == gradeable.getDependsOn() %} selected {% endif %}>{{ template.g_title }}-{{ template.g_id }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+        </fieldset>
+        <fieldset>
+            <legend>Points Required to Unlock:</legend>
```

Comment:
I think that it should be specified here that the points checked are only autograding points and not the overall score.